model/productsDao.js

```diff
@@ -24,12 +24,28 @@ const categoriesProducts = async (categoriesId) => {
   INNER JOIN sub_categories AS sc ON p.sub_category_id = sc.id
   INNER JOIN categories c ON sc.category_id = c.id
   WHERE c.id = ${categoriesId}
-  LIMIT 6
+  LIMIT 12
   OFFSET 0
   `);
 };
 
+const getCategoriesAllProducts = async (categoriesId, sort) => {
+  return await myDataSource.query(`
+    SELECT 
+      p.name AS productName,
+      p.thumnail_image_url AS thumbnailImageUrl,
+      p.short_description AS shortDescription,
+      p.price AS price
+    FROM products AS p
+    INNER JOIN sub_categories AS sc ON p.sub_category_id = sc.id
+    INNER JOIN categories c ON sc.category_id = c.id
+    ${sort}
+    ${categoriesId}
+    `);
+};
```

Comment:
아래와 같이 네이밍을 코드가 더 읽기 쉬워질 것 같습니다-!
```suggestion
const getProducts = async (whereClause, sort) => {
  return await myDataSource.query(`
    SELECT 
      p.name AS productName,
      p.thumnail_image_url AS thumbnailImageUrl,
      p.short_description AS shortDescription,
      p.price AS price
    FROM products AS p
    INNER JOIN sub_categories AS sc ON p.sub_category_id = sc.id
    INNER JOIN categories c ON sc.category_id = c.id
    ${sort}
    ${whereClause}
    `);
};
```